### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.14)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59481]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
When moving a message to another folder, the radio button group for selecting the destination folder doesn't have a `legend` to describe the radio options. When testing with NVDA and tabbing to the radio buttons, it's announced as an "undefined grouping." It looks like the `va-radio` component doesn't have a `label` property and has an `aria-label` of "undefined."

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Note that the design system's [guidance on radio buttons](https://design.va.gov/components/form/radio-button#accessibility-considerations) includes an instruction to have a `legend`, and the component in Storybook has the `label` property marked as required (which populates the `legend`).

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by lichellebain** [comment]
Jira ticket created:  https://jira.devops.va.gov/browse/MHV-46069
